src/datachain/lib/dc.py

```diff
@@ -859,6 +859,17 @@ def mutate(self, **kwargs) -> "Self":
                 dist=cosine_distance(embedding_text, embedding_image)
         )
         ```
+
+        This method also has enhanced capabilities for renaming the columns. If column
+        with the current name exists then its renamed. Orelse a new column is created
+        from the existing one.
+
+        Example:
+        ```py
+         dc.mutate(
+                oldkey=Column("newkey")
```

Comment:
my 2cs - I would make `newkey=Column("oldkey")` (since the intention here is to have a _new_ name).